docs/admin/performance_tuning.md

```diff
@@ -153,6 +153,74 @@ i2048 0      2Kb iovec Input only
     Number of repetitions, max inflight rpcs, message sizes can be adjusted based
     on the particular test/experiment.
 
+### Example: Without DAOS Agent
+
+For running `self_test` without a DAOS agent, the following environment variables needs to be
+defined:
+
+- `D_PROVIDER` defines mercury NA plugin and transport to be used (e.g.`ofi+verbs;ofi_rxm`)
+- `D_INTERFACE` defines network device name to be used (e.g. `ib0`)
+- `D_DOMAIN` defines the network name to be used (e.g. `mlx5_0:1`)
+
+The value of these environment variables could be found in the following log line of a DAOS agent
+daemon:
+
+```
+m02r01s01dao INFO 2025/02/17 17:17:27 pid:122802 (daos): numa:0 iface:ib0 dom:mlx5_0:1 prov:ucx+dc_mlx5 srx:1
+```
+
+A CaRT configuration file needs also to be provided to the `self_test` command.  This file should
+respect the following name pattern `<DAOS System Name>.attach_info_tmp` and respect the following
+format:
+
+- line 1: the DAOS System Name
+- line 2: The number of ranks
+- line 3: "all" or "self"
+  - "all" means dump all ranks' CaRT uri
+  - "self" means only dump this rank's CaRT uri
+- line 4 to #ranks: the list of ranks id and their CaRT uri (e.g. `ucx+dc_mlx5://10.6.4.104:32416`)
+
+An example of a file named `daos_server.attach_info_tmp` of a DAOS system named `daos_server` (i.e.
+default name of a DAOS system):
+
+```text
+name daos_server
+size 4
+all
+0 ucx+dc_mlx5://10.6.4.104:32416
+1 ucx+dc_mlx5://10.6.4.8:32416
+2 ucx+dc_mlx5://10.6.3.102:31416
+3 ucx+dc_mlx5://10.6.4.5:32416
+```
+
+This configuration file can be generated thanks to the `daos_agent` sub-command `dum-attachinfo`
+such as:
+
+```bash
+daos_agent -o daos_agent-self_test.yml dump-attachinfo -o daos_server.attach_info_tmp
+```
+
+This usage of the `daos_agent` command needs a minimal DAOS agent configuration file such as:
+
+```yaml
+name: daos_server
+access_points:
+- m02r01s01dao
+port: 10001
+transport_config:
+  allow_insecure: true
+log_file: /root/ckochhof/daos_agent-self_test.log
+```
+
+Pinging the SWIM service of the rank 0 without a DAOS agent could be done in the following way:
```

Comment:
rank 0, tag=1 to be precise, or you can change endpoint below to be just --endpoint 0 , instead of endpoint 0:1

Reply:
- [ ] Add tag ID of the SWIM service.

Reply:
> * [x]  Add tag ID of the SWIM service.

Not sure to have fully understand your comment.  It should be fixed with commit commit 5b9f9649f.